File: Sorting/SelectionSort.py
def Selection_sort(arr, n):
    for i in range (n-1):
        minIdx = i

        for j in range (i+1, n):
            if arr[j]<arr[minIdx]:
                minIdx = j
        
        arr[i], arr[minIdx] = arr[minIdx], arr[i]

    print(arr)

arr = [2, 5, 3, 6, 1]
Selection_sort(arr, len(arr))

File: Sorting/SelectionSort.java
package DSA.Sorting;

public class SelectionSort {
    public static void selectionSort(int[] arr, int n) {

        for (int i = 0; i < n - 1; i++) {
            int minIdx = i;
            for (int j = i + 1; j < n; j++) {
                if (arr[j] < arr[minIdx]) {
                    minIdx = j;
                }
            }
            int temp = arr[minIdx];
            arr[minIdx] = arr[i];
            arr[i] = temp;
        }
        for (int element : arr) {
            System.out.print(element);
        }
    }

    public static void main(String[] args) {
        int[] arr = { 2, 5, 3, 6, 1 };
        selectionSort(arr, arr.length);
    }
}
